python/dgl/graphbolt/impl/fused_csc_sampling_graph.py

```diff
@@ -1055,6 +1070,10 @@ def temporal_sample_neighbors(
             Boolean indicating whether the sample is preformed with or
             without replacement. If True, a value can be selected multiple
             times. Otherwise, each value can be selected only once.
+        input_nodes_time_window: torch.Tensor
+            Time window of the given seed nodes. The neighbors should be
```

Comment:
The neighbors or the edges?

Reply:
Both, will refactor the API to make it clear.